com.unity.sg2/Documentation~/previews/Arccosine.md

```diff
@@ -0,0 +1,11 @@
+## Description
+Calculates the arccosine of each component of the input.
+
+## Inputs
+**In** - input value
```

Comment:
nitpick - the input description could also have the first letter capitalized?

Reply:
I just updated all of my files so that lines start with a capital letter and end with a period.